Add expanded alt text for relevant images (#627)
Qiskit/documentation: docs/run/configure-error-mitigation.mdx

2-qubit gate layer.

<figure>

> <img src="/images/optimize/stratified.png" alt="Stratified circuit illustration. There are arbitrary 1-qubit gates between each `layer`. Each layer is defined by a block that crosses multiple qubit wires." />

Reviewer: ```suggestion
<img src="/images/optimize/stratified.png" alt="Stratified circuit illustration. There are arbitrary single-qubit gates between each `layer`. Each layer is defined by a block that crosses multiple qubit wires." />
```

illustrates the relationship between estimator error and number of
circuit samples for different total sampling overheads.

> ![This image shows that the error decreases a the number of samples increases.  The accuracy is best with a high sampling overhead (1000) and least with a low samping overhead (1.1).](/images/optimize/sampling-overhead.png)

Reviewer: ```suggestion
![This image shows that the error decreases as the number of samples increases.  The accuracy is best with a high sampling overhead (1000) and worst with a low sampling overhead (1.1).](/images/optimize/sampling-overhead.png)
```

multi-qubit Hilbert space, the resulting linear system of equations is
nominally much easier to solve.

> ![Illustration of the M3 method](/images/optimize/m3.png "M3 method")

Reviewer: ```suggestion
![Illustration of the M3 method.](/images/optimize/m3.png "M3 method")
```
